### PR TITLE
config default gwc config with jdbc/gwc

### PR DESCRIPTION
**Pull request by jeanmi151**
This need to wait for env var configuration in helm chart and docker deployments
connected with : https://github.com/georchestra/datadir/pull/340

### CODE REVIEW

File: gwc/geowebcache-diskquota-jdbc.xml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <gwcJdbcConfiguration>
   <dialect>PostgreSQL</dialect>
+  <JNDISource>jdbc/gwc</JNDISource>
```
Comment (fvanderbiest) [inline gwc/geowebcache-diskquota-jdbc.xml:3]: I see there's still the connectionpool info below.
This is a priori not required anymore with the JNDISource

Comment (jeanmi151) [reply gwc/geowebcache-diskquota-jdbc.xml:3]: I took exemple from geoserver configured with gwc jndi source and it keep the information after jndisource 
but i can remove it if it is working without

Comment (fvanderbiest) [reply gwc/geowebcache-diskquota-jdbc.xml:3]: okay

**Comment by fvanderbiest** [comment]
to be ported to branches `geofence`, `23.0` and `23.0-geofence`
Thanks !

**Comment by jeanmi151** [comment]
in geofence gwc folder config does not exist
same in 23.0-geofence
here in 23.0 : https://github.com/georchestra/geoserver_minimal_datadir/commit/550552da7969b4377f7f1b09f97c31dea0402bec

**Comment by fvanderbiest** [comment]
> in geofence gwc folder config does not exist
> same in 23.0-geofence

Woops, it should. But this is another story.

**Pull request merged**